src/index.js

```diff
@@ -1,9 +1,11 @@
 import assign from 'object-assign';
 import pascalCase from 'pascal-case';
+import React from 'react';
```

Comment:
What's your reasoning for avoiding peer dependencies? It kinda seems like we're doing the same roundabout thing, just with `window` or DI.